### ISSUE TITLE
enpkg mingw enstaller crashed

### ISSUE THREAD

**Issue by DennisMa820** [enthought/enstaller#15]
'enpkg mingw' command gives the follow result: 

enstaller: Error: enstaller crashed (uncaught exception <type 'exceptions.KeyError'>: Key Error('HTTP Error 401: UNAUTHORIZED: https://api.enthought.com/eggs/win-32/index.json?pypi=true',)).

**Comment by cournape** [comment]
Hi Dennis,

Could you give me the output of the following commands ?

```
$ enpkg --version
$ enpkg --config
```

**Comment by DennisMa820** [comment]
Hi David,

Here are the outputs:

enpkg --version
enstaller version: 4.6.5

enpkg --config
Python version: 2.7
enstaller version: 4.6.5
sys.prefix: C:/Users\onwer\AppData\Local\Enthought\Canopy32\User
platform: Windows-7-6.1.7600
architecture: 32bit
use_webservice: True
config file: C:\Users\onwer\.enstaller4rc
settings:
     prefix = C:\Users\onwer\AppData\Local\Enthought\Canopy32\User
     local =
'C:\\Users\\onwer\\AppData\\Local\\Enthought\\Canopy32\\User\\LOCAL-REPO'
     noapp = False
     proxy = None
     IndexedRepos: (not used)
          'https://www.enthought.com/repo/epd/GPL-eggs/Windows/x86/'
          'https://www.enthought.com/repo/epd/eggs/Windows/x86/'
          'http://www.enthought.com/repo/pypi/eggs/Windows/x86/'
Authentication failed: Authentication error: User login is required...
You are not logged in. To log in, type 'enpkg --userpass'.

I've tried the following

enpkg --userpass
Please enter the email address (or username) and password for your EPD or
EPD Free subscription. If you are not subscribed to EPD, just press Enter.

So you need to subscribe to EPD in order to install mingw?

**Comment by cournape** [comment]
Yes, the mingw package is part of the commercial offering. You are entitled to free access to the commercial packages if you are an academic, though.
